src/Razor/Microsoft.NET.Sdk.Razor/src/RazorGenerate.cs

```diff
@@ -125,6 +126,22 @@ protected override string GenerateResponseFileCommands()
                         builder.AppendLine(kind);
                     }
                 }
+
+                // Added in 5.0
+                if (parsedVersion.Major >= 5)
+                {
+                    // Most inputs won't have an associated CSS scope, so we only want to generate
```

Comment:
To make the command line arguments a little easier to understand for those looking at the ocmmand line lets re-iterate over Sources after this iteration is over and mention all of the css scopes & their input. That way it's extra clear that the previous command line parameters are 1-to-1. Alternatively we could just write out a cssscope for every item and have the other side of our system understand "empty" scopes mean nothing.